### DISCUSSION TITLE
How can I disable the blue overlay colour when using the navigation planning?

### DISCUSSION THREAD

**Discussion by Hund** [osmandapp/OsmAnd#19395] [Maps Related]
How can I disable the blue overlay colour when using the navigation planning? It makes things a lot harder to see.

<img src="https://github.com/osmandapp/OsmAnd/assets/320498/8f23f651-62c4-46e9-8211-579274384f1d)"  width="300" height="auto"/>

**Comment by dmpr0** [comment]
Hi! 
This one of the Weather layers, you can disable it in Menu – Configure Map – Weather.

**Reply by Hund** [reply]
Thank you! :D
